opal/core/search/templates/search.html

```diff
@@ -10,14 +10,15 @@ <h2>
       </h2>
     </div>
     <div class="panel-body">
-      <form ng-submit="search()" class="form form-horizontal">
+      <form ng-submit="!searching && search()" class="form form-horizontal">
         <div class="row content-offset">
             <div class="col-sm-4 col-sm-push-4">
               <input type="text" autofocus
                      name="name"
                      slash-key-focus="!state || state==='normal'"
                      ng-blur="enableShortcuts()"
                      ng-focus="disableShortcuts()"
+                     ng-change="searching=false"
```

Comment:
I think I want a comment here to highlight what this does - e.g. re-enables the search button if you change the search string.
I don't 100% know how to comment syntax inside indented single html element that won't break rendering/parsing